q2cli/core/cache.py

```diff
@@ -380,6 +381,8 @@ def _get_metavar(self, type):
             metavar = 'METADATA'
         elif style.style is not None and style.style != 'simple':
             metavar = 'VALUE'
+        elif qiime2.sdk.util.is_union(type):
+            metavar = ''
```

Comment:
I think this should be `VALUE` as well. I think this is similar to the immediately preceding branch in which it's just too complicated, so give up and say "put value here".